frontend/src/metabase/admin/settings/components/UploadSettings/UploadSettings.tsx

```diff
@@ -222,7 +224,7 @@ export function UploadSettingsView({
             <SectionTitle>{t`Upload Table Prefix`}</SectionTitle>
```

Comment:
In cases where the prefix is optional we should add `(Optional)` to the label here.

Reply:
👍 

![Screen Shot 2023-06-07 at 1 25 06 PM](https://github.com/metabase/metabase/assets/30528226/7f797ae6-083f-44fe-8da7-cdebccbcc326)
![Screen Shot 2023-06-07 at 1 24 55 PM](https://github.com/metabase/metabase/assets/30528226/463c0e80-8738-40bd-9e31-ec5558e04133)